modules/ocf_desktop/manifests/wireshark.pp

```diff
@@ -0,0 +1,15 @@
+class ocf_desktop::wireshark {
+  package {
+    'wireshark-common':
+      responsefile => '/var/cache/debconf/wireshark-common.preseed',
+      require      => File['/var/cache/debconf/wireshark-common.preseed'];
+
+    'wireshark':
+      ensure => present;
```

Comment:
if you want you can just do `'wireshark':;` since `ensure => present` is default